ember-cli-build.js

```diff
@@ -5,6 +5,9 @@ var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   var app = new EmberAddon(defaults, {
     // Add options here
+    babel: {
+      includePolyfill: true
```

Comment:
@stefanpenner Sorry should have been more explicit. I rapidly mentioned to you this issue at EmberConf. The problem here is that `null`/`undefined` exports are imported as complex objects (instead of `null`/`undefined`). You said that maybe you had an idea of what the problem/fix might be.